examples/ex1.cpp

```diff
@@ -176,31 +175,30 @@ int main(int argc, char *argv[])
    a->Assemble();
 
    Vector B, X;
-   Operator *A;
-   if (!pa) { A = new SparseMatrix; }
+   OperatorHandle Ah;
+   a->FormLinearSystem(ess_tdof_list, x, *b, Ah, X, B);
+   Operator &A = *Ah.Ptr();
 
```

Comment:
I like that this eliminates the pointer, but I still think this will be confusing for a lot of users...

Reply:
We can add a method `Ah.Oper()` or `Ah.Get()` that returns `Operator &` if `Ah.Ptr()` is confusing.

Reply:
I meant more: "why do we need two `Operator` objects and what is the difference between them?".